tests/aws/services/apigateway/test_apigateway_common.py

```diff
@@ -669,6 +683,48 @@ def test_create_update_stages(
         )
         snapshot.match("update-stage-override", response)
 
+    @markers.aws.validated
+    @markers.snapshot.skip_snapshot_verify(paths=["$..createdDate", "$..lastUpdatedDate"])
+    def test_update_stage_remove_wildcard(self, aws_client, _create_api_with_stage, snapshot):
+        client = aws_client.apigateway
+        api_id = _create_api_with_stage()
+
+        response = client.get_stage(restApiId=api_id, stageName="s1")
+        snapshot.match("get-stage", response)
+
+        def _delete_wildcard():
+            # remove all attributes at path */* (this is an operation Terraform executes when deleting APIs)
+            response = client.update_stage(
+                restApiId=api_id,
+                stageName="s1",
+                patchOperations=[
+                    {"op": "remove", "path": "/*/*"},
+                ],
+            )
+            snapshot.match("update-stage-reset", response)
+
+        # attempt to delete wildcard method settings (should initially fail)
+        with pytest.raises(ClientError) as exc:
+            _delete_wildcard()
+        snapshot.match("delete-error", exc.value)
```

Comment:
```suggestion
        snapshot.match("delete-error", exc.value.response)
```

not blocking, but this capture the proper exception structure

Reply:
Nice catch, 💯  thanks!

Reply:
might have to regenerate the snapshot though 😬